### DISCUSSION TITLE
Litecoin lightning on windows

### DISCUSSION THREAD

**Discussion by ECarline** [lightningnetwork/lnd#7714] [Q&A]
Is there any way to get lighning on windows currently, looked through bins and docs for compiling but couldn't find anything, the lightning explorer also seemed to be down

**Comment by ellemouton** [comment]
Hi @UpgradeDenied  - there are windows binaries available on the release page: https://github.com/lightningnetwork/lnd/releases

Heads up though that LND will likely be dropping Litecoin support soon.

**Comment by VovanichR** [comment]
yes this is good meaning
